src/main/java/Storage.java

```diff
@@ -0,0 +1,74 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+    // private static final String storagePath = "data/duke.txt";
+    private final String storagePath;
+
+    public Storage(String filePath) {
+        this.storagePath = filePath;
+    }
+
+    public void appendToFile(String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(storagePath, true);
+        fw.write(textToAdd);
+        fw.close();
+    }
```

Comment:
this function seems redundant given the writeToFile function. if you want flexibility, you could simply put an attribute in the class such that `true` means append, `false` means write, and make ur constructor take it in during instantiation.